awx/ui/client/src/users/users-tokens-add.controller.js

```diff
@@ -0,0 +1,99 @@
+function AddTokensController (models, $state, strings, Rest, Alert, Wait, GetBasePath, $filter) {
+    const vm = this || {};
+    const { application } = models;
+
+    vm.mode = 'add';
+    vm.strings = strings;
+    vm.panelTitle = strings.get('add.PANEL_TITLE');
+
+    vm.form = {};
+
+    vm.form.application = {
+        type: 'field',
+        label: 'Application',
+        id: 'application'
+    };
+    vm.form.description = {
+        type: 'String',
+        label: 'Description',
+        id: 'description'
+    };
+
+    vm.form.application._resource = 'application';
+    vm.form.application._route = 'users.edit.tokens.add.application';
+    vm.form.application._model = application;
+    vm.form.application._placeholder = strings.get('SELECT AN APPLICATION');
+    vm.form.application.required = true;
+
+    vm.form.description.required = false;
+
+    vm.form.scope = {
+        choices: ['', 'read', 'write'],
+        help_text: 'Specify a scope for the token\'s access',
+        id: 'scope',
+        label: 'Scope',
+        required: true,
+        _component: 'at-input-select',
+        _data: ['', 'read', 'write'],
+        _exp: 'choice for (index, choice) in state._data',
+        _format: 'array'
+    }
+
+    vm.form.save = data => {
+        Rest.setUrl(GetBasePath('users') + $state.params.user_id + '/authorized_tokens');
+        return Rest.post(data)
+            .then(({data}) => {
+                Alert('TOKEN INFORMATION', `
+                  <div class="TokenModal">
+                      <div class="TokenModal-label">
+                          TOKEN
```

Comment:
Do any of these strings (TOKEN, TOKEN INFORMATION) need to be translated? If so it might be worth it to go ahead and move these over to `tokens.strings.js`. I think some of these strings are actually already in there.